docs/threat-actor-intel.md

```diff
@@ -0,0 +1,70 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Threat Actor Intelligence
+
+The **Threat Actor Intelligence** feature in Threat Intelligence Management (TIM) provides analysts with detailed insights into adversaries, their motivations, and their tactics, techniques, and procedures (TTPs).
+
+## What is Threat Actor Intelligence
+
+A *threat actor* is an individual, group, or organization responsible for conducting malicious cyber activities. FortiGuard provides continuously updated profiles of threat actors that include their capabilities, targeted sectors, geographical scope, and known campaigns.
+
+By integrating this intelligence into TIM, SOC analysts gain enriched context that goes beyond indicators or feeds—linking observed activity to known adversaries and their historical behaviors.
+
+## How Threat Actor Data is Used in TIM
+
+- **Correlation with Threat Reports**: TIM automatically correlates FortiGuard threat actor data with existing feeds, reports, and ingested observables. Analysts can view connections between threat actors and ongoing campaigns, giving greater context to alerts.
+
+- **Mapping to MITRE ATT&CK**: Threat actor profiles are correlated with MITRE ATT\&CK data, updating TIM fields such as *techniques*, *sub-techniques*, and *tactics*. This mapping enables SOC analysts to quickly identify likely attack patterns and understand potential adversary objectives.
```

Comment:
The Threat actor also correlates with MITRE Group, please mention that as well

Reply:
Added. Thanks